core/build.gradle

```diff
@@ -37,23 +38,26 @@ android {
 dependencies {
     def filament_version = '1.12.0'
     def arcore_version = '1.26.0'
-    def appcompat_version = '1.3.1'
-    def material_version = '1.3.0'
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.0'
-    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.5.0'
-
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlin_version"
```

Comment:
this seems to have broken our android build since the kotlin verison is not the same as the coroutines version... ?

Reply:
It was a hard choice between kipping a global kotlin dependencies version to 1.5.0 or separate them because we have
```gradle
implementation "org.jetbrains.kotlin:kotlin-stdlib:1.53"
implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.0"
implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.5.2"
```
But it's true that Kotlin version is not completely linked to KotlinX versions

Anyway since they are declared as `implementation` this should not been an issue for you?
Is that's because you are using Sceneform as a module and share the 
```gradle
ext {
    kotlin_version = '1.5.30'
}
```
, I strongly recommend keeping Scenform as a separated project and using `publishToMavenLocal` and
```gradle
repositories {
    google()
    mavenLocal()
    mavenCentral()
}
```
within your app.

This way your `mavenLocal()` will be firstly used and you can test new releases by just increasing the version number without dependencies relations with your app.

Moreover if you have some fixes on Sceneform you will be able to easily create a Pull Request from Android Studio and share it with the community.

Reply:
thanks for your quick response. ah, yes we are using as a module due to earlier requiring a not yet released fix,
Should I revert both the core and ux to:
```
    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.0'
    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.5.0'
```

Reply:
go for
```gradle
ext {
    kotlin_version = '1.5.30'
}
```
and in build.gradle (:core)
```gradle
implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.0"
implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.5.2"
```
and PR it here.